client/src/components-questions/qsearch.jsx

```diff
@@ -1,16 +1,69 @@
 import React,{useState, useEffect} from 'react';
 import ReactDom from 'react-dom/client';
 import axios from 'axios';
-// import 'regenerator-runtime/runtime.js';
+import Qlist from './qlist.jsx';
+import Qadd from './addforms/qadd.jsx';
+import 'regenerator-runtime/runtime.js';
 // require("dotenv").config();
 
 
 
 var Qsearch = () =>{
+  const [term, setTerm] = useState('');
+  const[noQ,setNoQ]=useState(2);
+  const[questions,setQuestions]=useState([]);
+  const[initqs,setInitqs]=useState([]);
 
-  return (<div>
+  let load = (noQ,questions)=>{
+    if (noQ >= questions.length) {
+      return null;
+    } else {
+      return <button data-testid="load" onClick={()=>{setNoQ(noQ+2)}}>LOAD MORE QUESTIONS</button>;
+    }
+  };
 
-    <form>
+  let search = (sterm) =>{
+    let newArray = initqs.filter((question)=>{
+      return question.question_body.includes(sterm);
+    });
+    setQuestions(newArray);
+  };
+
+  let handleChange = (e) =>{
```

Comment:
I suggest a more meaningful name than handleChange, it's likely already being renamed to "onClick" when it's used, so a hint as to what happens "onClick" in the render would be helpful